docs/server-startup.md

```diff
@@ -68,29 +68,20 @@ For example, in this case your config file would be located at ```%TABPY_ROOT%\t
 
 ### Command Line Arguments
 
-- To specify the *port* on which your server instance listens, set the ```-p``` command line argument as follows:
+- To specify the *config file* with which to configure your server instance, set the ```--config=*``` or ```-c=*``` command line argument as follows:

```

Comment:
Did we remove the ability to set port independently of changing it in the config and setting your config on startup?

Reply:
yes -we only parse for the config file.  see tabpy-server\tabpy_server\app\app.py

Reply:
We removed the port CLI arg with [this change](https://github.com/tableau/TabPy/commit/0cc57b9901be5829a1e574a98da37925387cf9b8#diff-d0486b0d7b181591f52bad014cd3a06f).